build_tools/github_actions/amdgpu_family_matrix.py

```diff
@@ -100,6 +100,14 @@
 
 # The 'nightly' matrix runs on 'schedule' triggers.
 amdgpu_family_info_matrix_nightly = {
+    "gfx110x": {
+        "windows": {
+            "test-runs-on": "windows-gfx110X-gpu-rocm",
+            "family": "gfx110X-dgpu",
+            "bypass_tests_for_releases": True,
+            "build_variants": ["release"],
+        },
+    },
     "gfx90x": {
```

Comment:
If keeping the new entry in `amdgpu_family_info_matrix_nightly`, keep the list sorted (insert between `gfx103x` and `gfx1150`)